Test failure in LifecycleMvcAutoConfigurationTests.environmentWebEndpointExtensionDisabled()

```
java.lang.IllegalStateException: java.lang.IllegalArgumentException: Unknown format 'CONSOLE_LOG_STRUCTURED_FORMAT_IS_UNDEFINED'. Values can be a valid fully-qualified class name or one of the common formats: [ecs, gelf, logstash]
	at org.springframework.boot.context.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:347)
	at org.springframework.boot.context.logging.LoggingApplicationListener.initialize(LoggingApplicationListener.java:298)
	at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEnvironmentPreparedEvent(LoggingApplicationListener.java:246)
	at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:223)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:180)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:173)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:133)
	at org.springframework.boot.context.event.EventPublishingRunListener.multicastInitialEvent(EventPublishingRunListener.java:136)
	at org.springframework.boot.context.event.EventPublishingRunListener.environmentPrepared(EventPublishingRunListener.java:81)
	at org.springframework.boot.SpringApplicationRunListeners.lambda$environmentPrepared$0(SpringApplicationRunListeners.java:64)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at org.springframework.boot.SpringApplicationRunListeners.doWithListeners(SpringApplicationRunListeners.java:118)
	at org.springframework.boot.SpringApplicationRunListeners.doWithListeners(SpringApplicationRunListeners.java:112)
	at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:63)
	at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:350)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:310)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:149)
	at org.springframework.cloud.autoconfigure.LifecycleMvcAutoConfigurationTests.getApplicationContext(LifecycleMvcAutoConfigurationTests.java:47)
	at org.springframework.cloud.autoconfigure.LifecycleMvcAutoConfigurationTests.beanNotCreated(LifecycleMvcAutoConfigurationTests.java:135)
	at org.springframework.cloud.autoconfigure.LifecycleMvcAutoConfigurationTests.environmentWebEndpointExtensionDisabled(LifecycleMvcAutoConfigurationTests.java:52)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.IllegalArgumentException: Unknown format 'CONSOLE_LOG_STRUCTURED_FORMAT_IS_UNDEFINED'. Values can be a valid fully-qualified class name or one of the common formats: [ecs, gelf, logstash]
	at org.springframework.boot.logging.structured.StructuredLogFormatterFactory.get(StructuredLogFormatterFactory.java:148)
	at org.springframework.boot.logging.logback.StructuredLogEncoder.createFormatter(StructuredLogEncoder.java:78)
	at org.springframework.boot.logging.logback.StructuredLogEncoder.start(StructuredLogEncoder.java:68)
	at org.springframework.boot.logging.logback.LogbackConfigurator.start(LogbackConfigurator.java:111)
	at org.springframework.boot.logging.logback.DefaultLogbackConfiguration.createAppender(DefaultLogbackConfiguration.java:149)
	at org.springframework.boot.logging.logback.DefaultLogbackConfiguration.consoleAppender(DefaultLogbackConfiguration.java:131)
	at org.springframework.boot.logging.logback.DefaultLogbackConfiguration.apply(DefaultLogbackConfiguration.java:86)
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.lambda$loadDefaults$0(LogbackLoggingSystem.java:241)
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.withLoggingSuppressed(LogbackLoggingSystem.java:481)
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.loadDefaults(LogbackLoggingSystem.java:230)
	at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithConventions(AbstractLoggingSystem.java:84)
	at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:61)
	at org.springframework.boot.logging.logback.LogbackLoggingSystem.initialize(LogbackLoggingSystem.java:195)
	at org.springframework.boot.context.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:332)
	... 23 more
```

--- a/spring-cloud-context/src/test/java/org/springframework/cloud/autoconfigure/LifecycleMvcAutoConfigurationTests.java
+++ b/spring-cloud-context/src/test/java/org/springframework/cloud/autoconfigure/LifecycleMvcAutoConfigurationTests.java
@@ -48,7 +48,6 @@ public class LifecycleMvcAutoConfigurationTests {
 	}
 
 	@Test
-	@Disabled("TODO: https://github.com/spring-cloud/spring-cloud-commons/issues/1520")
 	public void environmentWebEndpointExtensionDisabled() {
 		beanNotCreated("writableEnvironmentEndpointWebExtension", "management.endpoint.env.enabled=false");
 	}
@@ -66,7 +65,6 @@ public class LifecycleMvcAutoConfigurationTests {
 
 	// restartEndpoint
 	@Test
-	@Disabled("TODO: https://github.com/spring-cloud/spring-cloud-commons/issues/1520")
 	public void restartEndpointDisabled() {
 		beanNotCreated("restartEndpoint", "management.endpoint.restart.enabled=false");
 	}

